src/app/api/webhook/route.ts

```diff
@@ -0,0 +1,20 @@
+import { NextRequest, NextResponse } from 'next/server';
+import { burnDimoTokens } from '@/services/smartContract';
+
+// POST request handler
+export async function POST(req: NextRequest) {
+  try {
+    const { status, txHash } = await req.json(); // Parsing JSON from the request body
```

Comment:
the comment is unnecessary since it's self explanatory
```suggestion
    const { status, txHash } = await req.json();
```